### ISSUE TITLE
[Accessibility] Copilot generated commit message is not read aloud

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#204300]
Type: <b>Bug</b>

Tested on Windows:

1. Make any changes in a cloned git repo.

1. Load screen reader, such as NVDA or JAWS.

1. Focus in the source control view and click "Generate Commit Message with Copilot" button.

* Note: The generated commit message is not announced to screen reader. Please add alert.

VS Code version: Code - Insiders 1.87.0-insider (2af613979f646fc4dcebfeaedc7d14f138c7b072, 2024-02-01T23:51:02.394Z)
OS version: Windows_NT x64 10.0.22631
Modes: Restricted

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 2611)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.71GB (7.08GB free)|
|Process Argv|--crash-reporter-id b05b88e5-8894-4031-ae34-fa034ebddea9|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (35)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-openapi|42C|4.21.1
aiprm-lang|AIP|0.0.2
Bookmarks|ale|13.4.2
mermaid-markdown-syntax-highlighting|bpr|1.6.0
vscode-markdownlint|Dav|0.53.0
gitlens|eam|14.6.1
prettier-vscode|esb|10.1.0
codespaces|Git|1.16.9
remotehub|Git|0.62.0
vscode-pull-request-github|Git|0.81.2024020111
lilypond-syntax|jea|0.1.1
scheme|jea|0.2.0
better-cpp-syntax|jef|1.17.2
lilypond-snippets|lhl|0.1.1
python|ms-|2023.25.10221012
vscode-pylance|ms-|2023.12.103
jupyter|ms-|2024.1.2024011201
jupyter-keymap|ms-|1.1.2
remote-containers|ms-|0.339.0
remote-ssh|ms-|0.109.2024020215
remote-ssh-edit|ms-|0.86.0
remote-wsl|ms-|0.81.8
vscode-remote-extensionpack|ms-|0.25.0
azure-account|ms-|0.11.6
azure-repos|ms-|0.38.0
remote-explorer|ms-|0.5.2024011009
remote-repositories|ms-|0.40.0
remote-server|ms-|1.6.2024011109
vscode-github-issue-notebooks|ms-|0.0.130
vscode-node-azure-pack|ms-|1.2.0
vsliveshare|ms-|1.0.5905
vscode-xml|red|0.26.1
vscode-yaml|red|1.14.0
lua|sum|3.7.4
vscodeintellicode|Vis|1.2.30

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vscod805cf:30301675
vsaa593:30376534
py29gd2263:30784851
c4g48928:30535728
a9j8j154:30646983
962ge761:30841072
pythongtdpath:30726887
welcomedialog:30812478
pythonidxpt:30768918
pythonnoceb:30776497
asynctok:30898717
dsvsc013:30777762
dsvsc014:30777825
dsvsc015:30821418
pythontestfixt:30866404
pythonregdiag2:30926734
pyreplss1:30879911
pythonmypyd1:30859725
pythoncet0:30859736
pythontbext0:30879054
accentitlementst:30870582
dsvsc016:30879898
dsvsc017:30880771
dsvsc018:30880772
8082a590:30953407
edj9j583:30943796
fegfb526:30952798
7j2b6412:30951517
e3gdj431:30946825
ccp2r3:30950906

```

</details>
